src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -25,11 +25,27 @@ const ALL_RELATED_ELEMENTS_LINK: FatLinkType = new FatLinkType({
     diagram: null,
 });
 
+export interface PropertySuggestionParams {
+    id: string;
+    key: string;
+    properties: string[];
+    lang: string;
+}
+export type PropertySuggestionCall = (params: PropertySuggestionParams) => Promise<Dictionary<PropertyScore>>;
```

Comment:
I think `PropertySuggestionHandler` would be a better name for this type. (Because it isn't a call per se but a function to call to.)